tests/test_algo_common.py

```diff
@@ -381,3 +385,58 @@ def test_can_export_compressed_model_with_specified_domain_for_custom_ops(tmp_pa
             count_custom_ops += 1
 
     assert count_custom_ops == 4
+
+
+def charnge_compression_algorithms_order(config):
```

Comment:
please be more attentive! 
charnge -> change

Reply:
I can still see `charnge` here.